### ISSUE TITLE
Feature Request: Effects

### ISSUE THREAD

**Issue by erixom** [Hyrules/WinHue3#55]
I'd like it so I can have it run something like 'Red Alert' and have it change to Red but also flash the lights on and off. Thanks.

**Comment by Hyrules** [comment]
Yes this is a planned feature. I will add this eventually but I cannot tell when.

**Comment by MrHamaga** [comment]
Do you have a public road map or feature backlog?

**Comment by Hyrules** [comment]
No right now it's all in my head. I keep the posts feature request open here so I remember what to implement.
